ci-operator/config/openshift-eng/ocp-qe-perfscale-ci/openshift-eng-ocp-qe-perfscale-ci-main__gcp-4.17-nightly-x86.yaml

```diff
@@ -44,6 +44,7 @@ tests:
   timeout: 6h0m0s
 - always_run: false
```

Comment:
If we put a `cron`, let's remove the `always_run`. Helps keep us consistent even if it is cosmetic.